cli/js/permissions_test.ts

```diff
@@ -11,18 +11,31 @@ const knownPermissions: Deno.Permission[] = [
 ];
 
 for (const grant of knownPermissions) {
-  testPerm({ [grant]: true }, function envGranted(): void {
-    const perms = Deno.permissions();
-    assert(perms !== null);
-    for (const perm in perms) {
-      assertEquals(perms[perm], perm === grant);
-    }
+  testPerm({ [grant]: true }, async function envGranted(): Promise<void> {
+    const status0 = await Deno.permissions.query({ name: grant });
```

Comment:
Should we not just start using the global Permissions object?

Reply:
https://github.com/denoland/deno/issues/2767#issuecomment-521464761

Reply:
OK. Let's move `permissions` under `navigator` and other classes to global.

Reply:
Maybe leave it for separate PR since it is apparently controversial

Reply:
Sorry I misread the thread. OK. Let's keep it for now.